HexCtrl/src/HexUtility.cpp

```diff
@@ -245,4 +245,17 @@ namespace HEXCTRL::INTERNAL
 
 		return { stSysTime };
 	}
+
+	DWORD GetUserDefaultDateFormat()
+	{
+		DWORD dwDateFormat { };
+
+		//Determine current user locale specific date format. Default to UK/European if unable to determine
+		//See: https://docs.microsoft.com/en-gb/windows/win32/intl/locale-idate
+		if (!GetLocaleInfoEx(LOCALE_NAME_USER_DEFAULT, LOCALE_IDATE | LOCALE_RETURN_NUMBER,
+			reinterpret_cast<LPWSTR>(&dwDateFormat), sizeof(dwDateFormat)))
+			dwDateFormat = 1;
+
+		return dwDateFormat;
+	}

```

Comment:
I think it's redundant. 
It's really used in just one place, in `SetDateFormat`, we better simply integrate it there.

Reply:
Fixed.